sdk/src/main/java/com/horizen/transaction/BoxTransaction.java

```diff
@@ -43,6 +43,9 @@
     @JsonProperty("typeId")
     public abstract byte transactionTypeId();
 
+    @JsonProperty("version")
+    public abstract byte version();
```

Comment:
add `@Override` since method was declared in the parent class.
And add the same for `transactionTypeId`, pls. That was missed.

Reply:
check `messageToSign` method. It must consider `version`.

Reply:
Done.